src/main/java/com/example/backendspringboottechiteasycontroller/controllers/TelevisionController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.backendspringboottechiteasycontroller.controllers;
+
+import com.example.backendspringboottechiteasycontroller.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    List<Television> televisionDatabase = new ArrayList();
+
+//    @GetMapping("/filter")
+//    public ResponseEntity<Object> getTelevisions(@RequestParam String brand, @RequestParam Integer maxPrice){
+//
+//        return ResponseEntity.ok(/*returnList*/);
+//    }
+    @GetMapping("/televisions")
+    public ResponseEntity<String> getTelevisions()
+    {
+        return ResponseEntity.ok("televisionDatabase");
+    }
+    @GetMapping("/television/{id}")
+    public ResponseEntity<Television> getSpecificTelevision(@PathVariable Integer id)
+    {
+        Television television = televisionDatabase.get(id);
+        if (television != null) {
+            return ResponseEntity.ok(television);
+        } else {
```

Comment:
You can only get to this statement if you purposefully enter a null value into the database. If you try t get an index that is out of bounds for the arraylist, you'll get an IndexOutOfBoundsException. You've created an exceptionhandler for that, so if that happens, you'll return a 404 (not found) tot the client.